app/regengine_client.py

```
@@ -12,6 +16,14 @@
 
 DEFAULT_LIVE_INGEST_ENDPOINT = "https://www.regengine.co/api/v1/webhooks/ingest"
 
+# Env var used to share an HMAC secret with the RegEngine ingest service.
+# When set, every live ingest request is signed with HMAC-SHA256 over the
+# exact request body bytes and sent as `X-Webhook-Signature: sha256=<hex>`.
+# When unset, signing is skipped — RegEngine's _verify_webhook_signature
+# also no-ops when its WEBHOOK_HMAC_SECRET is unset, so this preserves the
+# pre-signing migration ramp on both sides.
+WEBHOOK_HMAC_SECRET_ENV = "REGENGINE_WEBHOOK_HMAC_SECRET"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "WEBHOOK_HMAC_SECRET_ENV"

(S105)